server/src/main/java/io/druid/server/initialization/ZkPathsConfig.java

```diff
@@ -80,6 +80,11 @@ public String getCoordinatorPath()
     return (null == coordinatorPath) ? defaultPath("coordinator") : coordinatorPath;
   }
 
+  public String getOverlordPath()
+  {
+    return defaultPath("overlord");
```

Comment:
i intentionally did not make the latch path configurable . IMO only base zookeeper path should be configurable but all the other locations inside that base path should be dictated by Druid. I am not sure if there is any value in making internal paths configurable.
My understanding is that existing paths were made configurable only to support migration and not that there is any use case for them to be really user configurable.